doc/news/_preparation_next_release.md

```diff
@@ -118,6 +118,14 @@ you up to date with the multi-language support provided by Elektra.
 - Introduced `KeySet::remove(Key)` and `KeySet::remove(String)`
 - Removed `KeySet::lookup(Key, int)` and `KeySet::lookup(String, int)` as well as accompanying flag definitions `KeySet::KDB_O_NONE`, `KeySet::KDB_O_DEL` and `KeySet::KDB_O_POP`. Please use `KeySet::lookup(Key)` and `KeySet::lookup(String)` instead. Instead of `KeySet::KDB_O_DEL`, please consider using `Key::release`. The proper replacement for `KeySet::KDB_O_POP` is `KeySet::remove(Key)` or `KeySet::remove(String)`.
 - Native library proxy interface `Elektra` is now package private (previously was public).
+- Added example Java JNI plugin `whitelist`
+- Changed `Key nextMeta()` to `Optional<Key> nextMeta ()` no longer throwing NoSuchElementException for non-exceptional behavior
+- Added support of binary valued keys:
+  - Renamed `KeyBinaryTypeNotSupportedException` to `KeyStringValueException`
+  - Introduced `KeyBinaryValueException`
+  - Improved `Key` test coverage
+  - Introduced `Key::getBinary()` and `Key::setBinary(byte[])`
```

Comment:
This should be probably directly after "Added support of binary valued keys:"?

Reply:
@markus2330 I do not understand what you mean. What do you mean by this?

Reply:
The sentence  "- Introduced `Key::getBinary()` and `Key::setBinary(byte[])`" seems to be the most important feature for "Added support of binary valued keys:", so I would write it first.

Reply:
I will reorder the affected lines of the release note in #4002